docs/lti-integration.md

```diff
@@ -1,58 +1,47 @@
-# LTI 1.3 Integration
+# LTI v1.3 Integration
```

Comment:
NIT: The spec doesn't actually have the 'v' in the version number. For consistency with the core spec, consider just using `LTI 1.3`